user-interface/src/lib/components/uswds/ButtonGroup.tsx

```diff
@@ -1,7 +1,7 @@
 import { Children, createElement, isValidElement, ReactElement } from 'react';
 import { UswdsButtonStyle } from './Button';
 
-export interface ButtonProps {
+export interface ButtonGroupProps {
```

Comment:
Let's make these types, rather than interfaces, as we touch components.

Reply:
Sure.